qiita_pet/templates/study_ajax/artifact_file_selector.html

```diff
@@ -96,8 +105,14 @@
       // If the count is correct, then it is always correct. If the count is not
       // correct, it is correct if and only if it is not required and has no
       // children
-      if (count_ok || (!is_required && num_children === 0)) {
-        mark_as_correct(this);
+
+      if ((count_ok || (!is_required && num_children === 0))) {
+        if ($("#artifact-type").val() === 'per_sample_FASTQ' && num_samples !== num_children) {
+          message = "per_sample_FASTQ expects the same number of samples and run_prefix.";
```

Comment:
Maybe a slightly altered message to be clearer:
per_sample_FASTQ expects one run_prefix per sample. Check for duplicate or missing run_prefix entries.